pkg/trace/remoteconfighandler/remote_config_handler.go

```diff
@@ -57,6 +53,14 @@ func (h *RemoteConfigHandler) Start() {
 	h.remoteClient.RegisterAPMUpdate(h.onUpdate)
 }
 
+func (h *RemoteConfigHandler) Stop() {
```

Comment:
I had forgotten to make this handler stoppable 😬 (not really sure how important it actually is in this case, but at least it looks consistent with the rest)